### PR TITLE
[various] Fix admin addons

### PR DESCRIPTION
**Pull request by MrPetovan**
Depends on https://github.com/friendica/friendica/pull/7116
From https://forum.friendi.ca/display/a006f0fd-755c-cfbb-458f-f28665634766

Some addons were wrongly using the `addon_settings` hook to show the node-wide addon settings to admins only in their user settings. This PR rectifies this sad state by using the proper `<addon>_addon_admin` functions that are effectively used to show node-wide addon settings in the admin addon detail page.

Thanks Adam Gaskins for the keen eye!

### CODE REVIEW

File: cookienotice/cookienotice.php [hunk 1]
```diff
@@ -5,87 +5,66 @@
...
-	Hook::register('addon_settings_post', __FILE__, 'cookienotice_addon_settings_post');
-}
-
-/**
- * cookienotice_uninstall
- * unregisters hooks
- * 
- * @return void
-*/
-function cookienotice_uninstall()
```
Comment (tobiasd) [inline cookienotice/cookienotice.php]: Why do you remove the `uninstall` routine?

File: cookienotice/cookienotice.php [hunk 2]
```diff
@@ -5,87 +5,66 @@
...
  * registers hooks
- * 
+ *
  * @return void
  */
 function cookienotice_install()
 {
 	Hook::register('page_content_top', __FILE__, 'cookienotice_page_content_top');
 	Hook::register('page_end', __FILE__, 'cookienotice_page_end');
-	Hook::register('addon_settings', __FILE__, 'cookienotice_addon_settings');
```
Comment (tobiasd) [inline cookienotice/cookienotice.php]: How are the settings now called if you remove the hook?

File: geonames/README.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ Use Geonames service to resolve nearest populated location for given latitude, l
 
 ## Installation
 
-Pre-requisite: Register a username at geonames.org and set in `config/addon.config.php`
+Pre-requisite: Register a username at https://www.geonames.org/login and set it in `config/addon.config.php`
```
Comment (tobiasd) [inline geonames/README.md]: I would prefer the link to the intro page of the service. The login page has no information about the service at all, and the link to the login page is spotted easily from the intro page.

File: cookienotice/cookienotice.php (hunk 1 above)
Comment (MrPetovan) [reply cookienotice/cookienotice.php]: Definitely, but the documentation is in the core repo, nothing I can do here.

File: cookienotice/cookienotice.php (hunk 2 above)
Comment (MrPetovan) [reply cookienotice/cookienotice.php]: Definitely.

File: cookienotice/cookienotice.php (hunk 1 above)
Comment (MrPetovan) [reply cookienotice/cookienotice.php]: Done: https://github.com/friendica/friendica/pull/7122

File: cookienotice/cookienotice.php (hunk 2 above)
Comment (MrPetovan) [reply cookienotice/cookienotice.php]: Done: https://github.com/friendica/friendica/pull/7122

File: cookienotice/cookienotice.php (hunk 1 above)
Comment (tobiasd) [reply cookienotice/cookienotice.php]: Thanks

**Pull request merged**